tests/unit/test_utils.py

```diff
@@ -66,3 +71,33 @@ def test_double_quotes_inside():
 
 VALUES ({TEST_VALUE_ESCAPED}, 'c')"""
     ), test_insert_query
+
+
+def test_check_if_empty_file(caplog):
+    with open("empty.csv", "w"):
+        pass
+
+    with caplog.at_level(logging.WARNING):
+        check_if_empty_file(path="empty.csv", if_empty="warn", file_extension=".csv")
+        assert "The input file is empty." in caplog.text
+    with pytest.raises(ValueError):
+        check_if_empty_file(path="empty.csv", if_empty="fail", file_extension=".csv")
+    with pytest.raises(SKIP):
+        check_if_empty_file(path="empty.csv", if_empty="skip", file_extension=".csv")
+
+    os.remove("empty.csv")
+
+
+def test_check_if_empty_file_one_column(caplog):
+    df = pd.DataFrame(columns=["_viadot_downloaded_at_utc"], index=[0])
+    df.to_csv("empty.csv", index=False)
+
+    with caplog.at_level(logging.WARNING):
+        check_if_empty_file(path="empty.csv", if_empty="warn", file_extension=".csv")
+        assert "The input file is empty." in caplog.text
+    with pytest.raises(ValueError):
+        check_if_empty_file(path="empty.csv", if_empty="fail", file_extension=".csv")
+    with pytest.raises(SKIP):
+        check_if_empty_file(path="empty.csv", if_empty="skip", file_extension=".csv")
+
+    os.remove("empty.csv")
```

Comment:
Same as above, we're only testing the CSV here but the Parquet and other file types are handled by a separate code path inside `check_if_empty_file()`, so they should all be tested.